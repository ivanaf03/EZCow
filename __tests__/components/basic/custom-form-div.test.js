import React from "react";

import { render } from "@testing-library/react-native";

import CustomFormDiv from "../../../components/basic/custom-text-div";

describe("CustomFormDiv", () => {
  it("renders correctly", () => {
    const tree = render(<CustomFormDiv>Test</CustomFormDiv>);
    expect(tree).toMatchSnapshot();
  });
});