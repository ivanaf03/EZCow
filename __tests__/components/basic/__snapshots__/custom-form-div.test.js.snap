// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomFormDiv renders correctly 1`] = `
<View
  style={
    [
      {
        "paddingBottom": 16,
        "paddingLeft": 16,
        "paddingRight": 16,
        "paddingTop": 16,
      },
      {
        "backgroundColor": "#272727",
      },
      {
        "borderBottomLeftRadius": 8,
        "borderBottomRightRadius": 8,
        "borderTopLeftRadius": 8,
        "borderTopRightRadius": 8,
      },
      {
        "borderBottomWidth": 2,
        "borderLeftWidth": 2,
        "borderRightWidth": 2,
        "borderTopWidth": 2,
      },
      {
        "borderBottomColor": "#ffffff",
        "borderLeftColor": "#ffffff",
        "borderRightColor": "#ffffff",
        "borderTopColor": "#ffffff",
      },
    ]
  }
>
  Test
</View>
`;
